src/macros/conditional.rs

```diff
@@ -0,0 +1,10 @@
+#[macro_export]
```

Comment:
I would try to avoid macros. Sometimes it is necessary to use them, but in this case, I think we could just use this:

```rust
let foo = condition.then(|| "bar".to_string());
```

See: https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=431a9488b17d6e61b39afca8df4ff06e